Jenkinsfile.backup

```diff
@@ -0,0 +1,43 @@
+def TIMER = "H/30 * * * *"
+def NODE = "atomic-jslave-autobrew"
+
+def dirs_to_backup = ["rdgo", "images" ];
+
+node(NODE) {
+    def par_stages = [:]
+    stage("Clean workspace") {
+       step([$class: 'WsCleanup'])
+    }
+    checkout scm
+    utils = load("pipeline-utils.groovy")
+    utils.define_properties(TIMER)
+
+    for (dir in dirs_to_backup) {
+        // Expand, but also avoid iteration+closure trap
+        def target = "/srv/rhcos/output/${dir}";
+        par_stages["backup-" + target] = { -> stage("Backup " + target) {
+    utils.inside_assembler_container("") {
+        stage("Sync In") {
+            withCredentials([
+                string(credentialsId: params.ARTIFACT_SERVER, variable: 'ARTIFACT_SERVER'),
+                sshUserPrivateKey(credentialsId: params.ARTIFACT_SSH_CREDS_ID, keyFileVariable: 'KEY_FILE'),
+            ]) {
+                sh "mkdir -p ${target}"
```

Comment:
How about keeping this under `$WORKSPACE` and use the `rsync_dir_in_dest()` from #211? I really like the move away from `/srv`.